crates/api_common/src/post.rs

```diff
@@ -77,7 +71,6 @@ pub struct GetPost {
 pub struct GetPostResponse {
   pub post_view: PostView,
   pub community_view: CommunityView,
-  pub moderators: Vec<CommunityModeratorView>,
```

Comment:
Should no longer be necessary, and front ends should no longer have to pass around lists of moderators and admins in their UIs.